.github/workflows/haskell-ci.yml

```diff
@@ -6,11 +6,11 @@
 #
 #   haskell-ci regenerate
 #
-# For more information, see https://github.com/haskell-CI/haskell-ci
+# For more information, see https://github.com/andreasabel/haskell-ci
```

Comment:
Are you expecting your changes to be upstreamed or are you going to maintain this fork?